docs/guide-events.md

```diff
@@ -0,0 +1,69 @@
+---
+id: guide-events
+title: Considerations for fireEvent
+---
+
+## Interactions vs. events
+
+Based on [the Guiding Principles](guiding-principles.md), your test should
+resemble how users interact with your code (component, page, etc.) as much as
+possible. With this in mind, you should know that `fireEvent` isn't _exactly_
+how the user interacts with your application, but it's close enough for most
+scenarios.
+
+Consider `fireEvent.click` which creates a click event and dispatches that event
+on the given DOM node. This works properly for most situations when you simply
+want to test what happens when your element is clicked, but when the _user_
+actually clicks your element, these are the events that are typically fired (in
+order):
+
+- fireEvent.mouseOver(element)
+- fireEvent.mouseMove(element)
+- fireEvent.mouseDown(element)
+- element.focus() (if that element is focusable)
+- fireEvent.mouseUp(element)
+- fireEvent.click(element) And then, if that element happens to be a child of a
+  `label`, then it will also move focus to the form control that the label is
+  labeling. So even though all you really are trying to test is the click
+  handler, by simply using `fireEvent.click` you're missing out on several other
+  potentially important events the user is firing along the way. Again, most of
```

Comment:
Could you put "Again" on a new line so it's more of a prominent summary?

Reply:
done